### ISSUE TITLE
Keras ProgBar is not working properly anymore with TF 1.5.0

### ISSUE THREAD

**Issue by DEKHTIARJonathan** [tensorflow/tensorflow#16538]
If you run the following code with TF 1.4.*, this code works fine. However, if you run it with the newly released TF 1.5.0, the output is abnormal:

```python
import time
import tensorflow as tf

progress_bar = tf.contrib.keras.utils.Progbar(target=600)

for _ in range(600):
    progress_bar.update(_)
    time.sleep(0.01)
```

##### In Jupyter NB:
![image](https://user-images.githubusercontent.com/10923599/35502790-65112c36-04de-11e8-980f-59f1bc375d60.png)

**Comment by drpngx** [comment]
Thank you for the bug report!

@fchollet

**Comment by frankgh** [comment]
@fchollet I have submitted a PR to fix this issue: https://github.com/tensorflow/tensorflow/pull/16634

**Comment by DEKHTIARJonathan** [comment]
I will try as soon as I can @frankgh 
Thanks for the help

**Comment by tensorflowbutler** [comment]
Nagging Assignee: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @fchollet: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by DEKHTIARJonathan** [comment]
Just for the record, this issue is still not fixed with TF 1.6.0

**Comment by tensorflowbutler** [comment]
Nagging Assignee @fchollet: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @fchollet: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @fchollet: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by DEKHTIARJonathan** [comment]
Fixed. The force argument is deprecated, unfortunately no warning is raised: f176a61
